mindsdb/integrations/handlers/lightwood_handler/lightwood_handler/functions.py

```diff
@@ -150,41 +149,32 @@ def run_learn(df: DataFrame, args: dict, model_storage) -> None:
     db.session.commit()
 
 
-def run_adjust(name, db_name, from_data, datasource_id, company_id):
-    # @TODO: Actually implement this
-    return 0
-
-
-@mark_process(name='learn')
-def run_update(predictor_id: int, df: DataFrame, company_id: int):
+@mark_process(name='adjust')
+def run_adjust(df: DataFrame, args: dict, model_storage):
     try:
-        predictor_record = db.Predictor.query.filter_by(id=predictor_id).first()
+        base_predictor_id = args['base_model_id']
+        base_predictor_record = db.Predictor.query.filter_by(
+            id=base_predictor_id,
+            status=PREDICTOR_STATUS.COMPLETE
+        ).first()  # TODO should this be last instead?
```

Comment:
should this be last instead?

Reply:
no, we are filtering by id, there should be only one record